doc/sphinxman/source/conda.rst

```diff
@@ -107,46 +125,154 @@ all the libraries it needs. Now you need only specify a scratch directory
+    >>> conda create -n p4env python=x.x psi4 psi4-deps -c psi4/label/devel -c psi4
+
+Activate environment and make the adjustments to :envvar:`PATH` and
+:envvar:`PSI_SCRATCH` that it suggests at the end. Test with ``psi4
+--test``. Configuration for this set-up is summarized at
+:ref:`faq:runfrombinary`.
+
+**Details:**
+
+* It is strongly recommended to place |PSIfour| into a conda
+  environment where its libraries can't interfere with other programs (on
+  Linux, |PSIfour| installs a non-default gcc 5.2) rather than the main
+  Anaconda or Miniconda environment. Hence the creation of the environment
+  above, but the environment name (:samp:`{p4env}` above) can be
+  substituted.
+
+* Only reason for ``psi4-deps`` package is to get the QC runtime
+  add-ons; could say ``dftd3 gcp v2rdm_casscf`` instead of ``psi4-deps``;
+  or leave them out if you don't want them.
+
+
+.. _`faq:updatepsi4`:
+
+How to update a Psi4 binary
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Conda command to update an existing |PSIfour| conda installation to the
+newest stable release (roughly annually).
+
+.. warning:: Until 1.1 is official, this pulls the July 2016 1.0 version. 1.0 and 1.1 really shouldn't be mixed.
+
+.. code-block:: bash
+
+    >>> conda update psi4 -c psi4
+
+    # if psi4 channel added,
+    >>> conda update psi4
+
+Conda command to update an existing |PSIfour| conda installation to the
+latest development head (roughly nightly).
+
+.. code-block:: bash
+
+    >>> conda update psi4 -c psi4/label/devel
+
+    # if add-on updates required, try
+    >>> conda update psi4 -c psi4/label/devel -c psi4
+
+
+.. _`faq:psi4deps`:
+
+How to use conda to compile Psi4 faster and easier
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: bash
+
+    # Linux or Mac
+    # substitute x.x by 2.7|3.5|3.6 for alternate python versions
+    >>> conda create -n p4deps python=x.x psi4-deps -c psi4
+
+Same for Mac/Linux. Substitute desired python version: 2.7, 3.5, 3.6. Fine
+to choose your own env name. Activate environment, ``source activate
+p4deps``.  Go to where you've cloned psi4. Execute ``psi4-path-advisor``.
+It gives you a basic cmake command covering python, sphinx, link-time qc
+addons, and run-time qc addons. There's a help menu -h that gives more
+info. There's other options that will also pre-configure compilers. For
+example, at GaTech ``psi4-path-advisor --intel`` works. On Macs with
+XCode, ``psi4-path-advisor --clang`` works. Just read the help. For DGAS,
+there's an ``--disable``, but I don't encourage it. It gives you a fully
+functional cmake command, but those are just setting up CMake cache
+|w---w| like the plugins you can always add your own CMake variables to
+the command.
+
+
 .. _`sec:condadetails`:
 
-Conda Proficients
-^^^^^^^^^^^^^^^^^
+What do the conda packages psi4 & psi4-deps and the installer psi4conda contain
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Cool.  Just noticed it when scrolling through and figured I'd point it out in case it's not deliberate.